server/views/pages/start/legal-sender-start-page.njk

```diff
@@ -8,7 +8,8 @@
 {% block header %}
   {% include "partials/cookies/cookie-banner.njk" %}
   {{ govukHeader({
-    homepageUrl: "#"
+    homepageUrl: "https://www.gov.uk",
+    useTudorCrown: true
```

Comment:
From GOV.UK v5.2.0 don't need the `useTudorCrown` now as it's the default ()https://github.com/alphagov/govuk-frontend/releases/tag/v5.2.0). Can just remove it next time though.